libkbfs/disk_block_cache_helper.go

```diff
@@ -37,6 +37,8 @@ type DiskBlockCacheMetadata struct {
 	TriggeredPrefetch bool `codec:"HasPrefetched"`
 	// whether the block's triggered prefetches are complete
 	FinishedPrefetch bool
+	// the last tag with which the block was marked
+	Tag string
```

Comment:
Wouldn't it be more space efficient to make this a `byte` enum or bitfield?

Reply:
I erred on the side of debugability.  Not sure how a bitfield would work in practice -- it seemed hard to make the upper layer generate a new one for every mark-and-sweep if it's anything but a random string.

Reply:
I guess it's not obvious to me what you plan the tags to look like. Any reason it can't just be a timestamp?

Reply:
Oops, sorry for merging this with this discussion still unresolved.  I used the FBOID debug tag as the marking tag in the next PR: https://github.com/keybase/kbfs/pull/1963/files#diff-f6470cb87de71869397b1e97dd1ee43dR1100

That way, it's easy to debug the DB given the logs.  A millisecond timestamp would probably work too, but wouldn't show up in the debug tags.  Let me know if you want me to change it before I merge #1963.

Reply:
FBOID + a counter? I guess that's short enough that it shouldn't be a big deal.